docs/ref/contrib/admin/index.txt

```diff
@@ -1487,6 +1487,15 @@ templates used by the :class:`ModelAdmin` views:
     to return the same kind of sequence type as for the
     :attr:`~ModelAdmin.list_filter` attribute.
 
+.. method:: ModelAdmin.lookup_allowed(lookup, value)
+
+    The ``lookup_allowed`` method is given a lookup path
+    (e.g. ``user__email``) and a value, and returns a boolean indicating
```

Comment:
I've expanded the documentation significantly in the next commit, let me know your thoughts :) Really appreciate your diligence Tim.